doc/articles/faq.md

```diff
@@ -286,3 +286,83 @@ Yes, make sure to use the following project definition:
     <Reference Include="System.Runtime.Serialization" />
 </ItemGroup>
 ```
+
+## Does Uno offer a TreeView control?
+
+It's in the UWP API, but [not implemented yet](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tree-view)
+
+## Is there a table of all UWP controls and their features compared to what's offered under Uno?
+
+https://github.com/nventive/Uno/blob/master/doc/articles/supported-features.md
+
+##  Is there an Uno template that is based on portable class library?
+
+https://github.com/nventive/Uno/pull/132
+
+## Is there a Visual Studio template for Uno that incorporates the Prism library?
+
+No, but this sample can serve as a base: https://github.com/nventive/Uno.Prism/tree/uno/Sandbox/Windows10/HelloWorld
+
+## I get errors when serializing Json in Uno Wasm
+
+If you are using JSON.NET, you need [this](https://github.com/nventive/Uno.Playground/blob/master/src/Uno.Playground.WASM/LinkerConfig.xml) 
+This file is referenced in the .csproj like [that](https://github.com/nventive/Uno.Playground/blob/master/src/Uno.Playground.WASM/Uno.Playground.WASM.csproj#L43)
+
+## Is NavigationView supported in Uno?
+
+It is indeed not implemented at the time, see [nventive/Uno#4](https://github.com/nventive/Uno/issues/4) for now
+
+## Is there any particular reason that Uno uses a shared project? and is it possible to use a netstandard project instead?
+
+The view layer needs to be in a shared project because it has native dependencies.
```

Comment:
You can add: _Shared projects are also used to work around the fact that Visual Studio is not able to selectively compile a single Target Framework when building a cross-targeted library. Using a shared project improves the build performance when debugging a single platform._